src/main.coffee

```diff
@@ -206,6 +211,8 @@ window.FormRenderer = FormRenderer = Backbone.View.extend
         skip_validation: @options.skipValidation
       },
       @options.saveParams,
+      # TODO(fuf): We should exclude the follow up form params from here as they
+      # are already part of the loadParams. This may be a breaking change.
```

Comment:
Does including them twice change the payload that is sent at all?

Reply:
We have the follow up form params included at both the`follow_up_form_params` level and the root level. You made me notice that the `follow_up_form_params` scoping might be overdoing it here. Right now we only have the `follow_up_form_id` and `initial_response_id` under follow up form params and those won't conflict with any other load params or save params. I'll remove that scope and then I can remove the `@followUpFormParams` duplicate call.

Reply:
Cool, the scope changes look good.